### PR TITLE
pull request for updated things

### PR DESCRIPTION
**Pull request by sauravkr827**

### CODE REVIEW

File: src/main/webapp/WEB-INF/pages/login.jsp [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+       <tr>
+           <td> <b id="sign">---- Sign in ---- </b></td>
+
+       </tr>
+
+       <tr>
+
+               <td>
+                   <input type='text' name='email' value='' placeholder="Email" id="email">
```
Comment (saurav6180) [inline src/main/webapp/WEB-INF/pages/login.jsp:90]: change here username istead of email

File: src/main/webapp/WEB-INF/pages/login.jsp [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+               <td>
+                   <input type='text' name='email' value='' placeholder="Email" id="username">
+               </td>
+           </tr>
+       <br>
+       <br>
+
+           <tr>
+               <td>
+                   <input type='password' name='password' placeholder="Password" id="password"/>
```
Comment (sauravkr827) [inline src/main/webapp/WEB-INF/pages/login.jsp:98]: instead by password  use pwd

Comment (saurav6180) [reply src/main/webapp/WEB-INF/pages/login.jsp:98]: ok i will

Comment (sauravkr827) [reply src/main/webapp/WEB-INF/pages/login.jsp]: ok  good

**Pull request merged**